crmsh/parse.py

```diff
@@ -16,6 +16,7 @@
 
 
 _NVPAIR_RE = re.compile(r'([^=@$][^=]*)=(.*)$')
+_NVPAIR_NO_EMPTY_RE = re.compile(r'([^=@$][^=]*)=([^\s]+)$')
```

Comment:
Just being paranoid :-), what if an empty value is quoted with `'` or `''` like:

crm configure property stonith-timeout=''
crm configure property stonith-timeout=""

Reply:
I think the empty value for any property is meaningless, so the result is the same:
```
crm(live/15sp2-1)configure# property stonith-timeout=""
ERROR: syntax in property: Unknown arguments: stonith-timeout= near <stonith-timeout=> parsing 'property stonith-timeout='
crm(live/15sp2-1)configure# property stonith-timeout=''
ERROR: syntax in property: Unknown arguments: stonith-timeout= near <stonith-timeout=> parsing 'property stonith-timeout='
```